src/flagStore.ts

```diff
@@ -159,4 +184,29 @@ export class FlagStore {
 			this.store.all(DATA_KIND, resolve);
 		});
 	}
+
+	async allFlagsMetadata(): Promise<Dictionary<FeatureFlag>> {
```

Comment:
I think this doesn't need to be a promise any more

Reply:
It gives an error without it:
```The return type of an async function or method must be the global Promise<T> type. Did you mean to write 'Promise<Dictionary<FeatureFlag>>'?ts(1064)```